### PR TITLE
Tower Of Hanoi - C++

### PR DESCRIPTION
**Pull request by kauravankit25**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: C-Plus-Plus/other/tower_of_hanoi.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    towerOfHanoi(n - 1, from_rod, aux_rod, to_rod);
+    cout << "Move disk " << n << " from rod " << from_rod <<
+         " to rod " << to_rod << endl;
+    towerOfHanoi(n - 1, aux_rod, to_rod, from_rod);
+}
+
+// Driver code
+int main()
+{
+    int n = 4; // Number of disks
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/tower_of_hanoi.cpp]: Take Dynamic input and display suitable message before & after it

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/other/tower_of_hanoi.cpp (hunk 1 above)
Comment (kauravankit25) [reply C-Plus-Plus/other/tower_of_hanoi.cpp]: Done!!

File: C-Plus-Plus/other/tower_of_hanoi.cpp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+{
+    int n; // Number of disks
+    cout << "Enter number of disks: ";
+    cin >> n;
+    
+    towerOfHanoi(n, 'A', 'C', 'B'); // A, B and C are names of rods
+  
+    cout << "All disks have been moved to another rod!!!";
+    return 0;
+}
```
Comment (vatsalkesarwani12) [inline C-Plus-Plus/other/tower_of_hanoi.cpp:33]: Add the sample input-output in comment

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/other/tower_of_hanoi.cpp (hunk 2 above)
Comment (kauravankit25) [reply C-Plus-Plus/other/tower_of_hanoi.cpp:33]: I have done that inside the function.

Comment (vatsalkesarwani12) [reply C-Plus-Plus/other/tower_of_hanoi.cpp:33]: keep that comment outside the code in the bottom of the source code

Comment (kauravankit25) [reply C-Plus-Plus/other/tower_of_hanoi.cpp:33]: Done

**Comment by siddharth25pandey** [comment]
@kauravankit25 Resolve the merge conflicts error

**Pull request closed without merging**